synapse/push/pusherpool.py

```diff
@@ -147,9 +148,20 @@ async def add_pusher(
                 last_stream_ordering=last_stream_ordering,
                 last_success=None,
                 failing_since=None,
+                enabled=enabled,
             )
         )
 
+        # Before we actually create the pusher, we check if the user already has one for
```

Comment:
The method call before this is called `create_pusher` which seems to contradict the word `create` in this sentence. Perhaps "commit", "store" or "persist" instead?

Reply:
Ah good point.